dbsync-central/release_stuff/scripts/release_info.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+# EIP RELEASES INFO
+#
+export RELEASE_NAME="EIP-AFTER REFACTORY 1.0.0.2"
```

Comment:
REFACTORY?

Reply:
Just a name I used for current release for senders. Renaming it now